test/controllers/TransferCompleteControllerSpec.scala

```diff
@@ -0,0 +1,21 @@
+package controllers
+
+import java.util.UUID
+
+import play.api.test.CSRFTokenHelper.CSRFRequest
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{GET, contentAsString, defaultAwaitTimeout}
+import util.FrontEndTestHelper
+
+class TransferCompleteControllerSpec extends FrontEndTestHelper {
+
+  "TransferConfirmationController GET" should {
```

Comment:
Should this read: "Transfer**Complete**Controller ..."

Reply:
It should yeah, cheers. I've changed it.